index.html

```diff
@@ -10,6 +10,12 @@
     out in the same tree and use relative links so that they'll work offline.
    -->
 
+  <script src="https://code.jquery.com/jquery-3.6.0.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script>
+
+  <link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/1.10.24/css/jquery.dataTables.css">
+  
+  <script type="text/javascript" charset="utf8" src="https://cdn.datatables.net/1.10.24/js/jquery.dataTables.js"></script>
+  
```

Comment:
Alas! I do not think this is acceptable. A document in the W3C's /TR place cannot be dependent on an external library like jQuery (are we sure that this will still work in a few years?).

Also... I expect jQuery to clash with respec in some way or other. None of the features (getting to the next block, sorting) works for me in Brave (which is Chromium based), on my iPad (mobile Safari based), or Firefox as far as the ['preview' version](https://pr-preview.s3.amazonaws.com/jricher/did-spec-registries/pull/273.html#did-methods) goes.

Alas! our hands are tied. 

I did experiment with a much simpler solution, namely a relatively simple and standalone script that can either be added to the HTML code itself, or stored locally and linked to. That mechanism works, but it was unusable because very slow. We would have to find a very well optimized sorting script which can then be incorporated.

Reply:
Just to be on the safe side, though, I am happy to ask the powers to be whether it is acceptable to use jQuery after all. Indeed, W3C maintains on its own site, some 'frozen' copy of frameworks (the version of jQuery that is currently stored is 2.2.4). I do not know whether it is acceptable to use them in a /TR document.

@deniak @plehegar your input would be essential here.

Reply:
That being said, even if it is possible to use jQuery (or a specific version thereof) whether that can properly coexist with respec is a problem by itself. I may be wrong, but I seem to remember that jQuery does things on load-time, and so does respec. This means two dominating guys facing one another in the same bar...

Reply:
@iherman yes, we allow the use of scripts that are located under https://www.w3.org/scripts/. Now, we cannot ensure this won't conflict with respec and this is really a case by case basis

Reply:
> @iherman yes, we allow the use of scripts that are located under https://www.w3.org/scripts/. Now, we cannot ensure this won't conflict with respec and this is really a case by case basis

Ah. This may make things a bit better. Thanks @deniak 

@jricher you should try to use  https://www.w3.org/scripts/jquery/2.2/jquery.min.js as the reference to jquery, to see if it works. This resolves to the 2.2.4 version.

@deniak if we badly need a newer version (I have no idea what the status of jQuery is these days) then, I presume, that can also be installed on our side, right?

Reply:
> @deniak if we badly need a newer version (I have no idea what the status of jQuery is these days) then, I presume, that can also be installed on our side, right?

That should be doable. If you need a new version on /scripts, please send an email to sysreq.

Reply:
If I read all of the above correctly, `jquery.min.js` is not enough.  We also need DataTables to be added to the https://www.w3.org/scripts/ space... as we cannot refer to these external files (https://cdn.datatables.net/1.10.24/css/jquery.dataTables.css and https://cdn.datatables.net/1.10.24/js/jquery.dataTables.js).  Once that's done, things should work fine, as the current DataTables `*  @requires jQuery 1.7+`

Reply:
@TallTed , I just added datatables to https://www.w3.org/scripts/. Let me know if that solves the issue.

Reply:
> I expect jQuery to clash with respec in some way or other.

Although we'll try to support this in https://github.com/w3c/respec/issues/3483, I want to clarify that `respec-w3c` doesn't include jQuery, so there should be no clash. Though, to avoid any possible clash during the processing, consider enabling DataTable during [`postProcess`](https://respec.org/docs/#postProcess).